modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path

```diff
@@ -35,6 +35,16 @@
 	<td>//*[contains(@class,'lexicon-icon-plus')]//ancestor::button[contains(text(),'${key_buttonText}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TEXT_BUTTON</td>
+	<td>//h5[contains(text(),'${key_fieldText}')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @juliosilveira77, could you please update this path to: `//div[@role='tablist']//h5[contains(text(),'${key_textButton}')]`? I think that the way that you made is too generic.